pom.xml

```diff
@@ -73,17 +73,15 @@
 
     <dependency>
         <groupId>org.springframework</groupId>
-        <artifactId>spring-mock</artifactId>
-        <version>2.0.8</version>
+        <artifactId>spring-test</artifactId>
+        <version>5.2.8.RELEASE</version>
         <scope>test</scope>
     </dependency>
     <dependency>
         <groupId>org.springframework</groupId>
-        <artifactId>spring-core</artifactId>
-        <version>4.3.19.RELEASE</version>
-        <scope>core</scope>
+        <artifactId>spring-web</artifactId>
+        <version>5.2.8.RELEASE</version>
     </dependency>
-
```

Comment:
Please undo this change.

This change is good, but it's not part of the frontend work you're doing in this Pull Request. Instead, please put this change in a separate pull request, the one that you're doing on Java backend.